docsteady/spec.py

```diff
@@ -168,6 +168,32 @@ def process_steps(self, test_script):
         return teststeps_sorted
 
 
+def get_lvv_details(key):
+    """
+    get LVV information from Jira
+    :param key: LVV jira Key
```

Comment:
Thanks for adding docstrings, but you should try to use the official format https://developer.lsst.io/python/numpydoc.html instead of using something else.